performance_manager/thing_to_run.py

```diff
@@ -1,52 +1,108 @@
 #!/usr/bin/env python
 
-import pandas as pd
-import numpy as np
-import boto3
-import time
-import pathlib
-import pickle
-import psycopg2
-import os
+import argparse
 import logging
+import os
+import sys
 
-DBNAME=os.environ['DB_NAME']
-HOST = os.environ['DB_HOST']
-PASSWORD=os.environ['DB_PASSWORD']
-PORT=os.environ['DB_PORT']
-USER=os.environ['DB_USER']
-
-def get_aws_connection():
-    # TODO - figure out how to connect to the AWS RDS for Writing
-    #
-    # https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.Connecting.Python.html
-
-    #gets the credentials from .aws/credentials
-    session = boto3.Session()
-    client = session.client('rds')
-
-def get_local_connection():
-    """get a connection to the locally running postgres db"""
-    return psycopg2.connect(
-        host=HOST,
-        dbname=DBNAME,
-        user=USER,
-        port=PORT,
-        password=PASSWORD
-    )
+from sqlalchemy import insert
+from sqlalchemy.orm import sessionmaker
 
+from lib import (
+    SqlBase,
+    StaticSubHeadway,
+    get_experimental_engine,
+    get_local_engine,
+    process_static_schedule,
+)
 
-def main():
+logging.getLogger().setLevel("INFO")
+DESCRIPTION = """Entry Point to RDS Manipulating Performance Manager Scripts"""
+
+
+def load_environment() -> None:
+    """
+    boostrap .env file for local development
+    """
     try:
-        logging.info("Connecting to %s", DBNAME)
-        connection = get_local_connection()
-        cur = connection.cursor()
-        cur.execute("""SELECT now()""")
-        query_results = cur.fetchall()
-        logging.info(query_results)
+        if os.environ.get("BOOTSTRAPPED", 0) == 1:
+            return
+
+        here = os.path.dirname(os.path.abspath(__file__))
+        env_file = os.path.join(here, "..", ".env")
+        logging.info("bootstrapping with env file %s", env_file)
+
+        with open(env_file, "r", encoding="utf8") as reader:
+            for line in reader.readlines():
+                line = line.rstrip("\n")
+                line.replace('"', "")
+                if line.startswith("#") or line == "":
+                    continue
+                key, value = line.split("=")
+                logging.info("setting %s to %s", key, value)
+                os.environ[key] = value
+
     except Exception as exception:
-        logging.error("Database connection failed")
+        logging.error("error while trying to bootstrap")
         logging.exception(exception)
+        raise exception
+
+
+def parse_args(args: list[str]) -> argparse.Namespace:
+    """parse args for running this entrypoint script"""
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+    parser.add_argument(
+        "--experimental",
+        action="store_true",
+        dest="experimental",
+        help="if set, use a sqllite engine for quicker development",
+    )
+
+    return parser.parse_args(args)
```

Comment:
right now its just choosing which engine to use. i'm still not sure what the best way to do this is still.

it seems like the engine should just be an attribute of the lib itself but then its hard to mess with from the command line.